README.md

```diff
@@ -101,6 +101,39 @@ If your environment requires a proxy, define the proxy properties in the JSON fi
   "proxy": "<proxy_host>:<proxy_port>"
 ```
 
+## Exception handling
+
+All the exceptions raised by the OneView inherits from HPOneViewException.
+
+**HPOneViewException** have the following properties:
+- **msg**  has the message string error, extracted from message attribute from OneView response;
+- **oneview_response** has the entire JSON data (as dict) returned by the OneView; it can be ```None```.
```

Comment:
I suggest something like:
**oneview_response** has the entire JSON data (as dict) returned by the OneView, describing the specific problem with the request. It can be ```None```.